datalad/crawler/nodes/tests/test_misc.py

```diff
@@ -41,12 +44,62 @@
 @skip_if_no_network
 @use_cassette('brain-map.org-1', return_body='')
 def test_get_disposition_filename():
-    input = {'url': 'http://human.brain-map.org/api/v2/well_known_file_download/157722290'}
-    output = list(get_disposition_filename(input))
+    inputted = {'url': 'http://human.brain-map.org/api/v2/well_known_file_download/157722290'}
```

Comment:
`input` becomes underlined in red for being a built-in name to explain my edit. I switched it back though!